doc/dbus-protocol.txt

```diff
@@ -53,6 +53,8 @@ The following command require a successful CreateComparison in advance.
 method GetFiles config-name number1 number2 -> list(filename status)
 
 
+Intentionally not documented are PrepareQuota and QueryQuota.
```

Comment:
Until the implementation (including kernel) has shown to be satisfying I don't want to document the methods so that changing them is possible.